### PR TITLE
Added new asset id

### PR DESCRIPTION
**Pull request by ksr30**

### CODE REVIEW

File: primitives/src/assets.rs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use codec::{Decode, Encode};
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_core::{RuntimeDebug, H160};
+
+#[derive(Encode, Decode, Copy, Clone, PartialEq, Eq, Ord, PartialOrd, RuntimeDebug)]
```
Comment (Gauthamastro) [inline primitives/src/assets.rs:24]: Why is this here?

we moved the dependency, please remove the crate

**Review by Gauthamastro — CHANGES_REQUESTED** [review]

File: primitives/src/assets.rs (hunk 1 above)
Comment (ksr30) [reply primitives/src/assets.rs]: done

**Review by Gauthamastro — APPROVED** [review]
